### PR TITLE
Update "Dim (and brighten) lights via a remote" cookbook example

### PR DESCRIPTION
**Pull request by pnbruckner**
## Proposed change

Update cookbook example using new script features.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_cookbook/dim_and_brighten_lights.markdown [hunk 1]
```diff
@@ -89,79 +85,63 @@ input_number:
...
+                brightness: >
+                  {% set br = state_attr(light, 'brightness')|int(0) %}
+                  {% set mn = states('input_number.light_minimum')|int %}
+                  {% set mx = states('input_number.light_maximum')|int %}
+                  {% set st = states('input_number.light_step')|int %}
+                  {% if direction == 'up' %}
+                    {{ [br + st, mx]|min }}
+                  {% else %}
+                    {{ [br - st, mn]|max }}
+                  {% endif %}
```
Comment (frenck) [inline source/_cookbook/dim_and_brighten_lights.markdown:143]: Lets replace this with `brightness_step` or `brightness_step_pct`. There is no reason to calculate it anymore.

Comment (pnbruckner) [reply source/_cookbook/dim_and_brighten_lights.markdown:143]: Ah, good point. I've never used it, but now that you mention it, I do remember hearing about it.

Comment (pnbruckner) [reply source/_cookbook/dim_and_brighten_lights.markdown:143]: No, wait. This example uses `input_number`'s to limit the min & max values, so it does still have to calculate the new values anyway to be able to make sure those limits are not exceeded. If the example didn't do that then, yes, using `brightness_step` would be more optimal.

Comment (frenck) [reply source/_cookbook/dim_and_brighten_lights.markdown:143]: Alright!

**Review by frenck — APPROVED** [review]
Thanks 👍

**Pull request merged**